experiments/benchmarks/utils/benchmark/benchmarker.py

```diff
@@ -102,13 +102,14 @@ def set_benchmark_grid_config(self, config=None, dimensions=None, n_batch=None,
         }
         if config is not None:
             config = Configuration.from_yaml(config, check=False)
-
         self.set_benchmark_grid_config_param(benchmark_config, "dimensions", dimensions, config)
         self.set_benchmark_grid_config_param(benchmark_config, "keep_history", keep_history, config)
         # the base_integrand_params is too problem specific to be provided by direct argument/CLI.
         # Either a default or a config-file input
         self.set_benchmark_grid_config_param(benchmark_config, "base_integrand_params", None, config)
         self.set_benchmark_grid_config_param(benchmark_config, "base_integrator_config", None, config)
+        self.set_benchmark_grid_config_param(benchmark_config["base_integrator_config"], "flow", None, config)
+        self.set_benchmark_grid_config_param(benchmark_config["base_integrator_config"]["flow_options"]["cell_params"], "n_bins", None, config)
```

Comment:
These two lines are workarounds for a bug I introduced. Please merge the latest master branch and rebase the commits from this PR, you should be able to remove these two lines and still have the flow and flow options be set.